backend/tests/conftest.py

```
@@ -0,0 +1,50 @@
+import pytest
+from sqlalchemy.orm import Session
+from fastapi.testclient import TestClient
+import sys
+from pathlib import Path
+from fastapi import Depends
```

Ruff (0.8.2)

`fastapi.Depends` imported but unused

Remove unused import: `fastapi.Depends`

(F401)